src/components/Header/index.tsx

```diff
@@ -4,7 +4,7 @@ import logo from "../../assets/logo.svg";
 
 function Header() {
   return (
-    <Box paddingTop={10} className="App-header">
+    <Box paddingTop={10} className="App-header" marginBottom={10}>
```

Comment:
this was the other part of Custom Container. This marginBottom pushes the stuff under down so we don't need to account for that in the individual components